### ISSUE TITLE
Can't find the BRD2704A bootloader in bootloader_binaries_2.2.0-1.2.zip.

### ISSUE THREAD

**Issue by lboue** [SiliconLabs/matter#139]
### Description of the issue
Hi,

I can't find the BRD2704A / Sparkfun Thing Plus MGM240P bootloader in bootloader_binaries_2.2.0-1.2.zip.

![image](https://github.com/SiliconLabs/matter/assets/938089/c0b3cdb4-4a25-4246-a7e5-bf8cc5b42ae4)

### Steps to reproduce the issue

```bash
1) Download https://github.com/SiliconLabs/matter/releases/download/v2.2.0-1.2/bootloader_binaries_2.2.0-1.2.zip
2) Unzip bootloader_binaries_2.2.0-1.2.zip
3) Go to bootloader_binaries_2.2.0-1.2 directory


BRD2704A / Sparkfun Thing Plus MGM240P
```

### Device type(s) and network topology

BRD2704A / Sparkfun Thing Plus MGM240P board

### SMG version

release_2.2.0-1.2

### Protocol(s)

Thread

### Hardware platform(s)

BRD2704A

**Comment by lboue** [comment]
Same issue with :
 * [bootloader_binaries_1.1.0-1.1.zip](https://github.com/SiliconLabs/matter/releases/download/v1.1.0-1.1/bootloader_binaries_1.1.0-1.1.zip)
 * [bootloader_binaries_2.1.0-1.1.zip](https://github.com/SiliconLabs/matter/releases/download/v2.1.0-1.1/bootloader_binaries_2.1.0-1.1.zip)

**Comment by jmartinez-silabs** [comment]
fwiw, you can find it here
https://github.com/SiliconLabs/sdk_support/tree/main/matter/efr32/bootloader_binaries

But we should add it to the artifacts also.

**Comment by lboue** [comment]
> But we should add it to the artifacts also.

Feel free to post a message here when this is fixed.

**Comment by silabs-sebastien** [comment]
@lboue The binaries are now available in the release files as expected for 2.3.0-1.3-alpha.2 and beta.1.

**Comment by lboue** [comment]
Thanks.
